package.json

```diff
@@ -1,30 +1,26 @@
 {
-  "name": "nvquicktheme",
-  "version": "2.0.0",
-  "description": "Barebones Bootstrap 4 DNN Theme",
-  "main": "gulpfile.js",
-  "author": "nvisionative",
-  "license": "GPL-3.0",
-  "private": true,
+  "name": "40FINGERS-DNN-Theme",
+  "version": "1.0.0",
+  "description": "40FINGERS DNN Theme",
+  "main": "default.aspx",
   "dependencies": {
-    "bootstrap": "^4.6.2",
-    "font-awesome": "^4.7.0",
-    "normalize.css": "^8.0.1"
+    "@fortawesome/fontawesome-free": "^5.1.0-9",
```

Comment:
I am not familiar with this version.  What is the purpose for `^5.1.0-9` instead of just `^5.15.4`?

Reply:
@david-poindexter Well, I wondered about that too.
I followed the intructions on this page:

https://fontawesome.com/v5/docs/web/setup/use-package-managers

This is what's installed, not sure why.
I uninstalled and installed with the version number now.

Reply:
Awesome - looks good now.  Thanks!